Сделана разметка секции hero

RomanKorotaev opened the pull request:
Сделана разметка секции hero

RomanKorotaev committed:
Сделана разметка секции hero

Roman-Lahoda commented on src/partials/hero.html:
```diff
@@ -0,0 +1,31 @@
+<h1>Donut Masterclass</h1>
+<h1>with the Donut King</h1>
```
заголовок h1 должен быть один на странице, для правильного переноса слов можно воспользоваться  <br>, но лучше просто ограничить его по ширине чтобы автоматом переносилось

Roman-Lahoda replied:
заголовок h1 должен быть один на странице, для правильного переноса слов можно воспользоваться br, но лучше просто ограничить его по ширине чтобы автоматом переносилось

Roman-Lahoda commented on src/partials/hero.html:
```diff
@@ -0,0 +1,31 @@
+<h1>Donut Masterclass</h1>
+<h1>with the Donut King</h1>
+<p>Hosted by Eva Green</p>
```
в ТЗ написали что "Hosted by Eva Green" подзаголовок, поэтому лучше в тег h2 обернуть чтобы на проверки у ментора не возникало вопросов

Roman-Lahoda commented on src/partials/hero.html:
```diff
@@ -0,0 +1,31 @@
+
+<ul>
+    <li>
+        <a href="#">
```
всем ссылкам которые ведут на внешние ресурсы желательно добавить атрибуты target="_blank" и rel="nofollow noreferrer"

RomanKorotaev committed:
Внесены правки в разметку, согласно замечаний

Приметка:  подзаголовок с текстом "Hosted by Eva Green"  обвёрнут в тег <h2>.  Ментор на вебинаре говорила, что лично она обвернула бы этот подзаголовок в тег <p>. Но, думаю это не критично и можем выбрать такой тег, который тут захотим использовать.

RomanKorotaev committed:
Update hero.html
